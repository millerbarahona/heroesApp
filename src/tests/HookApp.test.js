import React from "react";
import { shallow } from "enzyme";
import { HeroesApp } from "../HeroesApp";
import {setupTest} from './setupTests';

setupTest(); //se hace para poder llamar la configuracion de test

describe('pruebas en HookApp', () => {
  test('debe mostrarse bien', () => {
    const wrapper = shallow(<HeroesApp />);
    expect(wrapper).toMatchSnapshot();
  });
});