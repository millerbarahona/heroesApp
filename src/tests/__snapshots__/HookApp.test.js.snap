// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en HookApp debe mostrarse bien 1`] = `
<ContextProvider
  value={
    Object {
      "dispatch": [Function],
      "user": Object {
        "logged": false,
      },
    }
  }
>
  <AppRouter />
</ContextProvider>
`;
